paper-slider.html

```diff
@@ -90,6 +90,10 @@
       :host(:focus) {
         outline: none;
       }
+        
+      :host-context([dir="rtl"]) {
```

Comment:
Is `:host:dir(rtl)` supported yet? I still see `:host-context` used elsewhere, such as in 
https://github.com/PolymerElements/paper-checkbox/blob/master/paper-checkbox.html.

Is it reasonable to continue to use `:host-context([dir="rtl"])` for the time-being (until a replacement is available)